CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,219 @@
+
+**Response and Potential Consequences**
+
+The Quansight COC Committee will recommend a response to the Quansight Labs Co-Directors (or higher up the Quansight leadership team if appropriate).
+
+Potential consequences (for reportees) of Quansight Code of Conduct incidents include:
+
+* Nothing (if it is determined that no breach occurred)
+* Private feedback or reprimand from Quansight to the reportee(s) involved
+* Warning the reportee to cease their behavior and that any further incident reports will result in sanctions
+* A public announcement that an incident occurred
+* Mediation (only if both reporter(s) and reportee(s) agree)
+* A permanent or temporary ban from some or all Quansight spaces (mailing lists, GitHub repos, in-person events, etc.)
+* Assistance to the complainant with a report to other bodies (e.g., institutional offices or appropriate law enforcement agencies)
+* An imposed vacation or removal from Quansight employment/affiliation (for Quansight personnel/affiliates)
+* Publishing an account of the incident and calling for the resignation of the reportee(s) from their responsibilities (usually pursued when Quansight lacks formal authority: this may be appropriate if the person is an external event leader, if they refuse to recuse themselves from a conflict of interest, or similar)
+* Any other response that Quansight deems necessary and appropriate to the situation
+* At Quansight events, if a participant engages in behavior that violates this Code of Conduct, the organizers and staff may take any action they deem appropriate
+
+At an in-person Quansight-organized event, potential consequences for Code of Conduct incidents include:
+
+* Warning the offender to cease their behavior and that any further incidents will result in sanctions
+* Requiring that the offender avoid any interaction with—and physical proximity to—any relevant person(s) (e.g., in the case of harrassment) for the remainder of the event
+* Immediately ending all the offender's remaining event responsibilities and privileges
+* Expelling the offender from the event immediately and requiring that they do not return
+* Banning the person from future events (either indefinitely or for a certain time period)
+* Any other response that Quansight deems necessary and appropriate to the situation
+
+No one espousing views or values contrary to the standards of our Code of Conduct will be permitted to hold any position representing Quansight (volunteer positions included). Quansight has the right and responsibility to remove, edit, or reject comments, commits, code, edits, and other online contributions that are not aligned with this Code of Conduct.
```

Comment:
I'm not a fan of this sentence, and I imagine it could cause backlash if anyone bothers to read the CoC. I see that it was taken from the NumFOCUS CoC.

Particularly, the "espousing views or values" part sounds like this is about silencing dissent, whereas the rest of the CoC (rightly) is focused on behavior. Read literally, it can easily be interpreted as saying that no one is allowed to have any criticisms of any part of the CoC itself. Perhaps we can replace "espousing views or values" with "engaging in behavior" or some similar. 

Also, to be sure certain "espousing of views" can be harmful as behavior, but I think that these things are already described above, e.g., "Comments which have the effect of excluding people on the basis of membership in a specific group". So if anything, this text not only comes off the wrong way, but seems limiting to only those things that are representing a view or value, rather than what I think it intends to say, which is that people within Quansight shall not be permitted to violate the CoC.